spec/order_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "order"
+
+describe Order do
+  describe "#new_item" do
+
+    it "finds the ordered item from the menu and saves it into complete_order" do
+      order = Order.new
+      order.new_item("Duck_Pancakes", 1)
+      expect(order.complete_order).to eq([{ Dish: "Duck_Pancakes", Price: "12" }])
+    end
+
+    it "can add more than multiple quantites of an item to an order" do
+      order = Order.new
+      order.new_item("Duck_Pancakes", 2)
+      expect(order.complete_order).to eq([{ Dish: "Duck_Pancakes", Price: "12" }, { Dish: "Duck_Pancakes", Price: "12" }])
+    end
+  end
+
+  describe "#total" do
+
+    let(:food_options) { double :food_options }
+    let(:text_class) { double :text_class, :confirmation => 0 }
+    let(:mock_printer) { double :mock_printer, :print_bill => 0 }
+    let(:mock_calculator) { double :mock_calculator, :calculate_total => "something" }
+
+    it "uses the calculator to get a total" do
+      order = Order.new(food_options, mock_calculator, mock_printer, text_class)
+      expect(order.total).to eq(order.calculator.calculate_total(order.complete_order))
+    end
+  end
+
+  describe "#print_order" do
+
+    let(:food_options) { double :food_options }
+    let(:text_class) { double :text_class, :confirmation => 0 }
+    let(:mock_printer) { double :mock_printer, :print_bill => 0 }
+    let(:mock_calculator) { double :mock_calculator, :calculate_total => "something" }
+
+    it "sends an order and bill to the printer" do
+      order = Order.new(food_options, mock_calculator, mock_printer, text_class)
+      expect(order.print_order).to eq(order.print_class.print_bill(order.complete_order, order.total))
```

Comment:
It's not clear what this method should return - would be better to specify the expected value.